### DISCUSSION TITLE
[Bug]: PWA manifest - use relative path for start URL

### DISCUSSION THREAD

**Discussion by kade-d** [blakeblackshear/frigate#13603] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

When frigate is served at a subdirectory like `https://server.com/frigate` and it is installed as a PWA, launching the PWA opens the base URL `https://server.com` instead of the subdirectory `https://server.com/frigate`.

I believe this can be fixed by using a relative path like `./` for the start_url instead of `/` in the web manifest

### Steps to reproduce

1. Host frigate at a subfolder, e.g., `https://server.com/frigate`
2. Open frigate in iOS at `https://server.com/frigate` and select "Add to Home Screen" in Safari or Chrome
3. Open the PWA that was installed at the iOS home screen
4. You will be taken to `https://server.com` instead of `https://server.com/frigate`

### Version

0.14.1-f4f3cfa

### In which browser(s) are you experiencing the issue with?

Google Chrome: 128.0.6613.120

### Frigate config file

```yaml
N/A
```

### docker-compose file or Docker CLI command

```yaml
N/A
```

### Relevant Frigate log output

```shell
N/A
```

### Relevant go2rtc log output

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Network connection

Wired

### Camera make and model

N/A

### Screenshots of the Frigate UI's System metrics pages

N/A

**Comment by NickM-27** [comment]
I'll have to double check but I believe doing this caused other issues

**Reply by kade-d** [reply, marked as answer]
I'm unable to host frigate at a subdomain for reasons out of my control, but I ended up just hosting frigate at a specific port instead of a subdirectory to work around this so not an issue for me anymore
